src/main/java/com/volunteer/enterprise/VolunteerAppController.java

```diff
@@ -54,23 +76,24 @@ public Volunteer createVolunteer(@RequestBody Volunteer volunteer){
         try {
             newVolunteer = volunteerService.save(volunteer);
         }catch (Exception e){
-            //TODO later
+            e.printStackTrace();
```

Comment:
I prefer the TODO over the printStackTrace.

- Logging is the best approach... but, we haven't discussed logging yet.
-  TODO will be flagged by the IDE until that time.
- See notes above.  try-catches shouldn't be a syntax must-have, but rather, a way to stop and handle anything that goes wrong, to give our users a better experience.